packages/aws-cdk-lib/aws-ec2/lib/volume.ts

```diff
@@ -70,6 +70,14 @@ export interface EbsDeviceOptionsBase {
    * `@aws-cdk/aws-ec2:ebsDefaultGp3Volume` is enabled.
    */
   readonly volumeType?: EbsDeviceVolumeType;
+
+  /**
+   * The throughput that the volume supports, in MiB/s
+   * Takes a minimum of 125 and maximum of 1000.
+   * @see https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/EBSVolumeTypes.html
+   * @default - 125 MiB/s. Only valid on gp3 volumes.
```

Comment:
```suggestion
   * The throughput to provision for a `gp3` volume.
   *
   * Valid Range: Minimum value of 125. Maximum value of 1000.
   *
   * @see https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/EBSVolumeTypes.html
   *
   * @default - 125 MiB/s.
```

Reply:
For this and for your other code feedback below, I have copied this verbatim from the existing code for autoscaling. For example: https://github.com/aws/aws-cdk/blob/da2ec75f6bc8f2e28b07cdf1307b22c83bb652d5/packages/aws-cdk-lib/aws-autoscaling/lib/auto-scaling-group.ts#L2430-L2451

Should we keep these consistent or?

Reply:
I'd go for the cleaner version and provide some documentation about the `0.25` ratio if possible, thanks.